addons/website/static/src/snippets/s_rating/001.scss

```diff
@@ -13,3 +13,12 @@
         }
     }
 }
+
+.s_rating_active_icons,.s_rating_inactive_icons {
+    margin: 0!important;
+    padding: 0!important;
+    .fa {
+        margin: 0 0 0 .5em!important;
+    }
+}
```

Comment:
- Missing comment to understand why it is necessary. There already is no padding and margin on those elements :thinking: 
- Space before important
- Space after `,`
- This should be inside the `[data-vcss="001"]` environement if really needed.
- I doubt this is needed, the point it to have a natural spacing. If spaces are lost, it's a bug that should not be solved with CSS.

Reply:
yeah this thing it's what we kept after I discussed with bedo about the problem but it's only part of the original solution (that was involving quite a lot of modification in all example and some class switching in the corresponding js) I can remove it if needed
The thing is that the whitespaces are removed at some point (even with writing it as &nbsp) by the editor and thus only the spacing between the span stays at the end
The main problem with that is that you don't have the same result with the XML example (because the whitespaces are kept) and when you modify the number of active/inactive icons and when you save the modifications in addition to unconsistencies with the spacing between the icons